### PR TITLE
Rename to hugie 🐻

### LINKED ISSUE #19: Rename `hfie` to `hugie` 🐻

### PR DESCRIPTION
**Pull request by nsorros**
# Description

This renames the tool to hugie 🐻

⚠️ After the PR is approved the repo will be renamed and all previous links will redirect to the new one. Even though git commands will still work as per Githubs [documentation](https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository) it's better to either clone fresh or update the remote url `git remote set-url origin git@github.com:MantisAI/hugie.git`

Fixes #19 

# Checklist

- [x] Tests added
- [x] Relevant issue mentioned
- [x] Documentation updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
-
-To get started, you must set your individual or organisation Huggingface token into an env var called `HUGGINGFACE_READ_TOKEN`.
+⚠️  To get started, you must set your individual or organisation Huggingface token into an env var called `HUGGINGFACE_READ_TOKEN`.
 
-# Usage
+# Usage 📺
 
 tldr; watch the video:
 
 [![asciicast](https://asciinema.org/a/QA68DK9RPsNiO2zycx4ihugg0.svg)](https://asciinema.org/a/QA68DK9RPsNiO2zycx4ihugg0)
```
Comment (ivyleavedtoadflax) [inline README.md:21]: Need to redo this

File: setup.cfg [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [metadata]
-name = hfie
+name = hugie
 version = 0.1.0
 author = Matthew Upson
```
Comment (ivyleavedtoadflax) [inline setup.cfg:4]: Let's update this

**Review by ivyleavedtoadflax — APPROVED** [review]
Looks good, just a couple of minor things

File: setup.cfg (hunk 2 above)
Comment (nsorros) [reply setup.cfg:4]: What should we write?

File: README.md [hunk 3]
```diff
@@ -1,35 +1,31 @@
-# Huggingface Inference Endpoints helper
+# Hugie 🐻
 
-Documentation: https://mantisai.github.io/hfie/ 📑
+📖 Official [documentation](https://mantisai.github.io/hfie/)
```
[the remark this answers is not in the document]
Comment (nsorros) [reply README.md]: 8f42349

File: setup.cfg (hunk 2 above)
Comment (ivyleavedtoadflax) [reply setup.cfg:4]: I meant to add you as author

File: README.md (hunk 1 above)
Comment (ivyleavedtoadflax) [reply README.md:21]: Let's add an issue for it

File: setup.cfg (hunk 2 above)
Comment (nsorros) [reply setup.cfg]: 9f0017e

File: README.md (hunk 1 above)
Comment (nsorros) [reply README.md:21]: Cool 👍
